ui/src/app/components/sharded-dataflow-migration-details-form/sharded-dataflow-migration-details-form.component.scss

```diff
@@ -0,0 +1,74 @@
+.icon {
```

Comment:
Do we need specific CSS for this ? Can we not re-use the css for all forms ?

Reply:
I have largely followed the convention used in the existing code for this.
Ref - https://github.com/search?q=repo%3Acloudspannerecosystem%2Fharbourbridge%20.icon&type=code

Will look into what how we can re-use CSS for all forms.

Reply:
I checked this, the global CSS is present here - https://github.com/cloudspannerecosystem/harbourbridge/blob/master/ui/src/styles.scss

However, icons are currently styled differently in different areas - 
https://github.com/search?q=repo%3Acloudspannerecosystem%2Fharbourbridge%20.icon&type=code

In order to not cause cascading changes, I'll refrain from adding an `icon` styling in the global CSS for now.

Reply:
CSS can exist at different levels/files